README.md

```diff
@@ -56,7 +55,7 @@ REQUIRED SECTIONS:
 
```

Comment:
![image](https://github.com/otmaneechchafyky/bookstore/assets/93039359/0a10c9ff-71f5-46be-a527-d02b9b6f61ab)

**[MANDATORY]**

- [ ] Kindly fix your Live Demo link in the Table of contents because right now it is broken :fearful: Kindly fix it or remove it to make your readme descriptive and professional :blush: